spec/states/create_account_spec.rb

```diff
@@ -0,0 +1,70 @@
+RSpec.describe States::CreateAccount do
+  let(:context) { instance_double('Context', accounts: []) }
+  let(:state) { described_class.new(context) }
+  let(:name) { 'John Doe' }
+  let(:login) { 'johndoe' }
+  let(:password) { '123456' }
+  let(:age) { '32' }
+  let(:current_account) { instance_double('Account', name: name, login: login, password: password, age: age) }
+
+  describe 'when action' do
+    before do
+      allow(state).to receive(:name_input).and_return(name)
+      allow(state).to receive(:login_input).and_return(login)
+      allow(state).to receive(:password_input).and_return(password)
+      allow(state).to receive(:age_input).and_return(age)
```

Comment:
allow(state).to receive_messages(name_input: name, login_input: login, password_input: password, age_input: age). The same below